projects/kit/components/tabs/tabs-with-more.component.ts

```diff
@@ -231,20 +231,22 @@ export class TuiTabsWithMore implements AfterViewChecked, AfterViewInit {
 
         const {exposeActive, minMoreWidth} = this.options;
         const {clientWidth} = this.el;
-        const activeWidth = tabs[activeItemIndex] ? tabs[activeItemIndex].scrollWidth : 0;
-        const moreWidth = Math.max(tabs[tabs.length - 1].scrollWidth, minMoreWidth);
+        const activeWidth = tabs[activeItemIndex]
+            ? (tabs[activeItemIndex]?.scrollWidth ?? 0)
```

Comment:
We just checked that it exists, doesn't Typescript understand that? Let's change to:
```ts
const active = tabs[activeItemIndex];
const activeWidth = active?.scrollWidth ?? 0;
```

Reply:
fixed